examples/analysis/tabs/create_tables.jl

```diff
@@ -0,0 +1,24 @@
+# This scripts calls all scripts to create tables for comparing forecasts of the abm and the baseline models (AR, ARX, VAR, VARX)
+import BeforeIT as Bit
+using LaTeXStrings, CSV, HDF5, MAT
+
+# Clear old files
+foreach(rm, filter(endswith(".h5"), readdir("./examples/analysis/tabs/",join=true)))
+foreach(rm, filter(endswith(".tex"), readdir("./examples/analysis/tabs/",join=true)))
```

Comment:
```suggestion
foreach(rm, filter(endswith(".h5"), readdir("./",join=true)))
foreach(rm, filter(endswith(".tex"), readdir("./",join=true)))
```

for paths consistency